crates/accelerate/src/unitary_compose.rs

```diff
@@ -217,24 +197,25 @@ fn _ind(i: usize, reversed: bool) -> usize {
     }
 }
 
-/// For equally sized matrices, ``left`` and ``right``, check whether all entries are close
-/// by the criterion
-///
-///     |left_ij - right_ij| <= atol + rtol * right_ij
-///
-/// This is analogous to NumPy's ``allclose`` function.
-pub fn allclose(
+pub fn mm2q(
     left: &ArrayView2<Complex64>,
     right: &ArrayView2<Complex64>,
-    rtol: f64,
-    atol: f64,
-) -> bool {
-    for i in 0..left.nrows() {
-        for j in 0..left.ncols() {
-            if (left[(i, j)] - right[(i, j)]).abs() > atol + rtol * right[(i, j)].abs() {
-                return false;
+    qargs: &[Qubit],
+) -> Array2<Complex64> {
+    let mut out = arr2(&[
+        [C_ZERO, C_ZERO, C_ZERO, C_ZERO],
+        [C_ZERO, C_ZERO, C_ZERO, C_ZERO],
+        [C_ZERO, C_ZERO, C_ZERO, C_ZERO],
+        [C_ZERO, C_ZERO, C_ZERO, C_ZERO],
+    ]);
```

Comment:
Is there a reason to not do?:
```suggestion
    let mut out = Array2::zeros((4, 4));
```

Reply:
There's zero reason not to 😉